src/cargo.rs

```diff
@@ -50,7 +50,7 @@ impl Rustflags {
     pub fn encode(mut self, home: &Home) -> String {
         self.flags.push("--sysroot".to_owned());
         self.flags.push(home.display().to_string()); // FIXME: we shouldn't use display, we should keep the OsString
-        // As per CARGO_ENCODED_RUSTFLAGS docs, the separator is `0x1f`.
+                                                     // As per CARGO_ENCODED_RUSTFLAGS docs, the separator is `0x1f`.
         self.flags.join("\x1f")
```

Comment:
This formatting change looks semantically wrong. I think the first comment is about `home.display()...`, but the second one isn't a continuation of it, it's about `self.flags.join("\x1f")`. The way rustfmt will probably respect the old semantics is by having an empty line between them